src/components/App.js

```diff
@@ -1,14 +1,41 @@
 import React from 'react';
 import Display from './Display';
 import ButtonPannel from './ButtonPannel';
-// eslint-disable-next-line no-unused-vars
 import calculate from '../logic/calculate';
 
-const App = () => (
-  <div className="app m-auto">
-    <Display />
-    <ButtonPannel />
-  </div>
-);
+class App extends React.Component {
```

Comment:
- [ ] [OPTIONAL] You can also import component in the same line you get React from: 
```
import React, { Component } from 'react';

...

class App extends Component {
...
```